content/en/docs/components/pipelines/pipelines-quickstart.md

```diff
@@ -159,30 +85,20 @@ Follow these steps to set up the necessary GCP services and run the sample:
 1. Click the name of the run on the experiments dashboard.
 1. Explore the graph and other aspects of your run by clicking on the 
   components of the graph and the other UI elements. The following screenshot
-  shows the graph when the pipeline has finished running:
+  shows part of the graph when the pipeline has finished running:
     <img src="/docs/images/pipelines-xgboost-graph.png" 
       alt="XGBoost results on the pipelines UI"
       class="mt-3 mb-3 border border-info rounded">
 
-You can find the source code for the XGBoost training sample in the 
+You can find the source code for the **XGBoost - Iterative model training** demo in the 
```

Comment:
Suggested revision:

You can find the [source code for the **XGBoost - Iterative model training** demo](https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm) in the 
Kubeflow Pipelines repo.